### PR TITLE
Feature/behat filters test

### PR DESCRIPTION
**Pull request by tgrochowski**

### CODE REVIEW

File: src/Behat/Model/OperationRequest.php [hunk 1]
```diff
@@ -107,7 +107,29 @@ public function getFormatted(): array
...
+
+        if (count($filters) <= 0) {
+            return '';
+        }
+
+        $filtersData = '';
+
+        /**
+         * @var string $filterName
+         * @var mixed $filterValue
```
Comment (Codeweld) [inline src/Behat/Model/OperationRequest.php:120]: Do we need these comments?

File: src/Behat/Model/OperationRequest.php [hunk 2]
```diff
@@ -107,7 +107,29 @@ public function getFormatted(): array
...
+         */
+        foreach ($filters as $filterName => $filterValue) {
+            $filtersData .= sprintf(
+                "%s: %s,\n",
```
Comment (Codeweld) [inline src/Behat/Model/OperationRequest.php:124]: Should we add `,\n` also to the last one?

File: src/Behat/Model/OperationRequest.php (hunk 1 above)
Comment (tgrochowski) [reply src/Behat/Model/OperationRequest.php:120]: Psalm felt better with them ;)

File: src/Behat/Model/OperationRequest.php (hunk 2 above)
Comment (tgrochowski) [reply src/Behat/Model/OperationRequest.php:124]: GraphQL query allows leaving trailing comma

**Pull request merged**